addons/mail/static/src/js/followers.js

```diff
@@ -233,7 +235,13 @@ var Followers = AbstractField.extend({
         });
     },
     _reload: function () {
-        this.trigger_up('reload', {fieldNames: [this.name]});
+        if(this.postRefresh) {
```

Comment:
I'm not sure why this is needed, can you please explain? :-)

Reply:
When a new follower is added, it should be added in allowed user list as well. So this part is for the UI. Before this change, only the follower widget will be reloaded after a new follower is added. So for the user, they cannot see it is also added in the allowed user list in the UI (although in the backend, it is already added). And this change is to have an option to reload all components so the allowed user list will display properly.